tests/DataTransferObjects/LogLineTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Auditing\DataTransferObjects;
+
+use DateTime;
+use LoyaltyCorp\Auditing\DataTransferObjects\LogLine;
+use Tests\LoyaltyCorp\Auditing\TestCase;
+
+/**
+ * Class LogLineDtoTest
+ *
+ * @covers \LoyaltyCorp\Auditing\DataTransferObjects\LogLine
+ */
+class LogLineTest extends TestCase
+{
+    /**
+     * Test getters
+     *
+     * @return void
+     *
+     * @throws \Exception
+     */
+    public function testGetters(): void
+    {
+        $instance = $this->getInstance();
+        self::assertSame('127.0.0.1', $instance->getClientIp());
+        self::assertEquals(new DateTime('2019-05-20T11:12:13Z'), $instance->getOccurredAt());
+        self::assertSame(['customer_id' => 11], $instance->getRequestData());
```

Comment:
This may not have been entirely clear from the ticket, but Request/Response are strings (ie; the HTTP request & response heads & bodies)

Reply:
changed